### ISSUE TITLE
AZW cannot be DeDRMed? 

### ISSUE THREAD

**Issue by jl3807** [apprenticeharper/DeDRM_tools#1541]
I tried both Mobi and AZW but only AZW does not work. Is there a workaround for this? I'm using Calibre 5.11.0 and DeDRM 7.0.3. 
Thank you!

**Comment by ElleKayEm** [comment]
Yes, they can be.  See this post for details on getting files that can be decrypted:  https://www.mobileread.com/forums/showthread.php?t=283371

**Comment by jl3807** [comment]
> Yes, they can be. See this post for details on getting files that can be decrypted: https://www.mobileread.com/forums/showthread.php?t=283371

I tried method 1 and 2 and did not work, unfortunately. Thanks for your help though and I would appreciate it if you have another solution for me to attempt!

**Comment by ElleKayEm** [comment]
Should work with most purchased books and Kindle for PC/Mac on the same machine and user as calibre with DeDRM.  Let's see a log:  https://github.com/apprenticeharper/DeDRM_tools/blob/master/FAQs.md#i-cannot-solve-my-problem-with-the-dedrm-plugin-and-now-i-need-to-post-a-log-how-do-i-do-that
